packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -414,10 +415,11 @@ export function getInternalReactConstants(
       case IndeterminateComponent:
         return getDisplayName(resolvedType);
       case ForwardRef:
-        // Mirror https://github.com/facebook/react/blob/7c21bf72ace77094fd1910cc350a548287ef8350/packages/shared/getComponentName.js#L27-L37
```

Comment:
The previous behavior did not mirror the linked version of `getComponentName` since it didn't include `ForwardRef` if no `displayName` was set.